app/(main)/league/all/page.tsx

```diff
@@ -26,7 +28,11 @@ const Leagues = (): JSX.Element => {
     try {
       const userLeagues = await getUserLeagues(user.leagues);
       setLeagues(userLeagues);
-    } catch (error) {}
+    } catch (error) {
+      console.error(error);
```

Comment:
Throw an error if leagues are not retrieved.

Reply:
Added in console.error

Reply:
Added `throw new Error('Error fetching user leagues');`

Reply:
Removed the console.error